code/mathematical-algorithms/sum_of_digits/Sum_of_digits_function.py

```diff
@@ -1,11 +1,13 @@
 #Part of Cosmos by OpenGenus Foundation
 def sum_of_digits(number):
+    # converts the number into a string which is further converted into an array of numbers using map and 
+    # sum function calculating the sum of numbers in the list
     return sum(map(int, str(number)))
     
 try:
```

Comment:
If you want, you could remove the python two implementation altogether, since we are moving towards Python3 as per our style guide for Python.